EMIP/1-1000/EMIP00317.xml

```diff
@@ -39,23 +40,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   <summary>School Chants, የቃል፡ ትምህርት፡</summary>
                     <msItem xml:id="ms_i1"><locus from="1r" to="11v"/>
                     <textLang mainLang="gez">Gə‘əz</textLang>
+                    <title>Chants called Mästägabǝ’<foreign xml:lang="gez">መስተጋብዕ፡</foreign></title>
                     <incipit xml:lang="gez">ዕዝል: መስተጋብዕ። ዘሰኑይ: ወለቡ: ጽራኅየ: ወአጽምዓኒ: ቃለ ስእለትየ: ከመ: ወልታ: ሥሙር: ከለልከነ:</incipit>
-                    <note>Chants called Mästägabǝ’</note>
                   </msItem>
                   <msItem xml:id="ms_i2"><locus from="11v" to="17v"/>
+                  <title>Chants called Qalǝyä aṣmǝ`<foreign xml:lang="gez">ቃልየ፡ አጽምዕ፡</foreign></title>
                   <textLang mainLang="gez">Gə‘əz</textLang>
                   <incipit xml:lang="gez">ቃልየ: አጽምእ: እግዚኦ: ሃሌ: ሉያ: ሃሌ: ሉያ: ወለቡ: ጸራኅየ: (sic) ሃሌ: ሉያ: ሃሌ: ሉያ: ወአጽምዓኒ: ንጉሥየኒ: ወአምላኪየኒ:..</incipit>
-                  <note>Chants called Qalǝyä aṣmǝ`</note>
```

Comment:
Please use the BM system, here and below, https://betamasaheft.eu/Guidelines/?q=transliteration&id=transliteration-principles